opal/opal_hot_reloader.rb

```diff
@@ -1,5 +1,7 @@
 require 'opal_hot_reloader/reactrb_patches'
 require 'opal-parser' # gives me 'eval', for hot-loading code
+require 'opal-jquery'
```

Comment:
I'm reluctant to force an opal-jquery dependency on projects just for that, maybe we write a find equivalent